Nonlinear_Model_Predictive_for_WECCCOMP/wecSimInputFile.m

```diff
@@ -0,0 +1,153 @@
+%% Wave Class  
+%%%%%%%%%%%%%%%%%%%% No Wave   %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% waves = waveClass('noWave');
+%     waves.H = Hm0;                                      % Wave Height [m]
+%     waves.T = Tp;                                       % Wave Period [s]
+% 
+
+%%%%%%%%%%%%%%%%%%%% No Wave CIC %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% waves = waveClass('noWaveCIC');                         % Initialize waveClass
+%     waves.H = 0.0;                                      % Wave Height [m]
+%     waves.T = 0.0;                                      % Wave Period [s]
+    
+%%%%%%%%%%%%%%%%%%%% Regular Waves  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% waves = waveClass('regularCIC');                      % Initialize waveClass
+%     waves.H             = Hm0;                        % Wave Height [m]
+%     waves.T             = Tp;                         % Wave Period [s]
+%     waves.wavegauge1loc = [-1.70, 0];                   % Wave Gauge 1 x-location
+%     waves.wavegauge2loc = [-1.50, 0];                   % Wave Gauge 2 x-location
+%     waves.wavegauge3loc = [-1.25, 0];                   % Wave Gauge 3 x-location
+
+%%%%%%%%%%%%%%%%%%%% Irregular Waves  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+    waves = waveClass('irregular');                     % Initialize waveClass
+    waves.H             = Hm0;                          % Wave Height [m]
+    waves.T             = Tp;                           % Wave Period [s]
+    waves.spectrumType  = 'JS';                         % Specify Wave Spectrum Type
+    waves.freqDisc      = 'EqualEnergy';                % Uses 'EqualEnergy' bins (default) 
+    waves.gamma         = gamma;
+    waves.phaseSeed     = 3.3;                          % Phase is seeded so eta is the same    
+
+% %%%%%%%%%%%%%%%%%%% Custom Wave Elevation (eta) Import %%%%%%%%%%%%%%%%%%
+% waves = waveClass('etaImport');
+% switch(SeaState)
+%     case 4;        waves.etaDataFile = './waveData/waveProfile4.mat';
+%     case 5;        waves.etaDataFile = './waveData/waveProfile5.mat';
+%     case 6;        waves.etaDataFile = './waveData/waveProfile6.mat';
+%     otherwise;     disp('No wave data available');      return;
+% end
+
+%% Body Class
+%%%%%%%%%%%%%%%%%%% Float - 3 DOF   %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+body(1) = bodyClass('hydroData/wavestar.h5');       % Initialize bodyClass
+    body(1).mass                = 3.075;            % Define mass [kg]   
+    body(1).momOfInertia        = [0 0.001450 0];   % Moment of Inertia [kg*m^2]     
+    body(1).geometryFile        = 'geometry/Float.stl'; % Geometry File
+    body(1).linearDamping       = zeros(6);         % Linear Viscous Drag Coefficient
+    body(1).linearDamping(5,5)  = 1.8;              % Linear Viscous Drag Coefficient, determined From Experimetnal Tests
+%     body(1).nlHydro             = 1;                % This option integrates the pressure due to the instantaneous wave elevation and the instantaneous body position.

```

Comment:
@Liam-Guerrero Thank you for all of the changes thus far. I'd like you to ask if for this example you remove line 80 here `body(1).nlHydro` as the `.stl` mesh currently available is likely insufficient to run at this moment. This repository can be updated at a later time if this feature is implemented.